packages/react/src/components/DateTimePicker/DateTimePicker.jsx

```diff
@@ -682,6 +686,7 @@ const DateTimePicker = ({
           className={classnames(`${iotPrefix}--date-time-picker__menu`, {
             [`${iotPrefix}--date-time-picker__menu-expanded`]: isExpanded,
           })}
+          style={{ '--zIndex': zIndex }}
```

Comment:
we may want to spread the rest of the styles here in case people want to add additional styles via this prop.